vendors/Makefile

```diff
@@ -0,0 +1,14 @@
+# Maybe this should be a bash script.
+# This is however only ment to be run by developers.
+
+help:
+	@echo "make one of the following targets" 
+	@echo
+	@egrep "^[0-9a-zA-Z\-]+:$$" Makefile | grep -v "help:"
+
+update-preexec:
+	# - Maybe we should make this into a git-subtree(1) instead
```

Comment:
I think this comment is on point- we should start using git-subtree for this kind of stuff

Reply:
I might have another idea I want to try before resorting to subtrees as they can also get a bit messy to work with.